### ISSUE TITLE
Regex support for method names

### ISSUE THREAD

**Issue by ndrwrbgs** [csnemes/tracer#76]
In my scenario - it was tracing far too verbosely on some property getters/setters for our command line arguments object. Is there a way that I can direct in the config file to ignore certain method names -- e.g. what I'd like to do is something like

```xml
<NoTrace className="Arguments" methodName="get_.*" />
<NoTrace className="Arguments" methodName="set_.*" />
```

**Comment by ndrwrbgs** [comment]
It looks like https://github.com/csnemes/tracer/tree/pattern-matching will handle this

**Comment by csnemes** [comment]
Just released 3.0.0-beta (the pattern-matching branch). Give it a try.

**Comment by ndrwrbgs** [comment]
`traceProperties="false"` even did it for me, thanks :)
